docs/files.md

```diff
@@ -746,3 +747,35 @@ A file's watermark can be removed by calling
 ```js
 client.files.removeWatermark('67890', callback);
 ```
+
+Get Representation
+------------------
+
+A file's representation can be retrieved by calling
+[`files.getRepresentationInfo(fileID, representationTypes, options,
+callback)`](https://opensource.box.com/box-node-sdk/Files.html#getRepresentationInfo).
+You will be able to fetch a pdf representation, thumbnail representation, multi-page images
+representation, and fetching text representation.
+
+You can fetch different dimensions for the representation by calling
+```js
+client.files.getRepresentationInfo('67890', client.files.representation.IMAGE_LARGE, null, callback);
+```
+Additionally there are two header options you can include. set_content_disposition_type and set_content_disposition_filename.
+
+set_content_disposition_type must be set to either 'inline' to try
+and open the representation within the browser or 'attachment' to force the browser to download the
+representation. If not supplied the user-agent of the client is allowed to interpret that however
+it wants. To ensure consistency, we recommend supplying this parameter.
```

Comment:
If this is important to provide for consistency, we may want to set a sane default for this SDK.